spec/classes/init_spec.rb

```diff
@@ -415,7 +415,7 @@
       it do
         expect {
           should contain_class('ntp')
-        }.to raise_error(Puppet::Error, /ntp::peers must be a string or an array or an hash. Detected type is <boolean>./)
+        }.to raise_error(Puppet::Error, /ntp::peers must be a string or an array or an hash./)
```

Comment:
we could add detecion for other (unsupported) types as well, but I don't think that's worth the effort here.